catalogue/webapp/components/Work/Work.tsx

```diff
@@ -175,41 +176,49 @@ const Work = ({ work }: Props) => {
         )}
 
 {archivesPrototypeSidePanel && isInArchive ? (
-        <div className="container">
-          <div className="grid">
-            <div
-              className={classNames({
-                [grid({
-                  s: 12,
-                  m: 6,
-                  l: 5,
-                  xl: 5,
-                })]: true,
-              })}
-            >
-              <ArchiveTree work={work} withModal={false} />
+        <>
+          <div className="container">
+            <div className="grid">
+              <WorkHeaderPrototype work={work} childManifestsCount={childManifestsCount} />
```

Comment:
`WorkHeaderPrototype` is copy-pasted `WorkHeader` with tweaks made, because I didn't want to start adding more conditionals based on toggles to components that are in production.